queries.sql

```diff
@@ -48,4 +48,37 @@ SELECT COUNT(*) FROM animals WHERE escape_attempts < 1;
 SELECT AVG(weight_kg) FROM animals;
 SELECT neutered, MAX(escape_attempts) FROM animals GROUP BY neutered;
 SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY species;
-SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
\ No newline at end of file
+SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
+
+
+/* Day 3 */
+
+SELECT name FROM animals
```

Comment:
- [ ] we have to select all amimals , Kindly use `*`, ``SELECT * FROM animals``.